### DISCUSSION TITLE
Is a scoop-installed powershell used by shovel when upgrading scoop->shovel?

### DISCUSSION THREAD

**Discussion by david-bakin** [Ash258/scoop#200] [Q&A] [answered]
I have a scoop installation, with `pwsh` installed (currently: 7.1.4).  The shovel readme under "requirements" says:

> PowerShell 7 should be preferred when possible

Is this installation of powershell 7 - via scoop - usable/used by shovel subsequently when I follow the installation instructions to upgrade to shovel?  (Or do I need to install powershell 7 "officially", e.g., via its msi?)

Secondary question:

Even after exiting/reopening my command window (CMD.exe) I don't have a `shovel` command, only `scoop`. (Which is running shovel.)  Is that intended?

**Comment by Ash258** [comment, marked as answer]
> Is this installation of powershell 7 - via scoop - usable/used by shovel subsequently when I follow the installation instructions to upgrade to

No and yes.
If you use `shovel.cmd update pwsh` it will use powershell 5.
If you will execute it inside the powershell, which you want to update it will be stopped as you cannot update anything, which is running.
Use global/local installation combination or just powershell 5 to update pwsh

> Will they "come with" the shovel upgrade, or are they "abandoned" and I should install them again and delete the scoop folder?

I really do not understand this question. 

> Even after exiting/reopening my command window (CMD.exe) I don't have a shovel command, only scoop. (Which is running shovel.) Is that intended?

What if you would follow the installation steps which clearly tell you to run `checkup` command, which tell you what to do?

**Reply by david-bakin** [reply]
yeah that was a bogus question, I've deleted it.  But I am curious why I don't have a `shovel` command - no`shovel.cmd` anywhere that I see!

**Reply by david-bakin** [reply]
(And the reason I didn't try it first was I didn't want to screw up an existing installation before I knew the answer - but I did do it in a VM, which is why I deleted the question)

**Comment by ev-dev** [comment]
@Ash258 hey unfortunately even after performing the exact steps as listed in the install instructions (yes including checkup & opening new pwsh instance), I still don't get the `shovel` or `shovel.cmd` available and was hoping you could help users understand what the intended behavior is.

I assumed `shovel` would basically just be an alias to scoop after seeing how the upgrade from scoop was done but maybe can you please provide just a bit more detail?

Sorry to be lazy and not dig thru source files myselfm but it seemed like maybe other users were going to benefit from hearing answer from the maintainer, and so maybe best just to add quick blurb to the readme as well. Thanks for all your work!

**Reply by Ash258** [reply]
Properly configure your path to shims folder.

Next time you will reuse completely different discussion your ability to comment will be removed.
